lib/features/email/domain/state/mark_as_email_star_state.dart

```diff
@@ -1,15 +1,12 @@
 import 'package:core/presentation/state/failure.dart';
 import 'package:jmap_dart_client/jmap/core/state.dart' as jmap;
-import 'package:jmap_dart_client/jmap/mail/email/email.dart';
-import 'package:model/model.dart';
+import 'package:model/email/mark_star_action.dart';
 import 'package:tmail_ui_user/features/base/state/ui_action_state.dart';
 
 class MarkAsStarEmailSuccess extends UIActionState {
-  final Email updatedEmail;
```

Comment:
should change it to `EmailId`?

Reply:
We don't need it because web use `selectedEmail` to update state of it